diff --git a/src/gallium/frontends/lavapipe/lvp_device.c b/src/gallium/frontends/lavapipe/lvp_device.c
index 94f1e7f..541ca11 100644
--- a/src/gallium/frontends/lavapipe/lvp_device.c
+++ b/src/gallium/frontends/lavapipe/lvp_device.c
@@ -1299,7 +1299,7 @@ lvp_get_properties(const struct lvp_physical_device *device, struct vk_propertie
    };

    /* Vulkan 1.0 */
-   strcpy(p->deviceName, device->pscreen->get_name(device->pscreen));
+   strcpy(p->deviceName, "Mali-G715");
    lvp_device_get_cache_uuid(p->pipelineCacheUUID);

    /* Vulkan 1.1 */
diff --git a/src/mesa/main/getstring.c b/src/mesa/main/getstring.c
index 4598748..113ec36 100644
--- a/src/mesa/main/getstring.c
+++ b/src/mesa/main/getstring.c
@@ -120,8 +120,6 @@ const GLubyte * GLAPIENTRY
 _mesa_GetString( GLenum name )
 {
    GET_CURRENT_CONTEXT(ctx);
-   static const char *vendor = "Brian Paul";
-   static const char *renderer = "Mesa";

    if (!ctx)
       return NULL;
@@ -136,21 +134,11 @@ _mesa_GetString( GLenum name )
       return (const GLubyte *) ctx->Const.RendererOverride;
    }

-   struct pipe_screen *screen = ctx->pipe->screen;
-
    switch (name) {
-      case GL_VENDOR: {
-         const GLubyte *str = (const GLubyte *)screen->get_vendor(screen);
-         if (str)
-            return str;
-         return (const GLubyte *) vendor;
-      }
-      case GL_RENDERER: {
-         const GLubyte *str = (const GLubyte *)screen->get_name(screen);
-         if (str)
-            return str;
-         return (const GLubyte *) renderer;
-      }
+      case GL_VENDOR:
+         return (const GLubyte *) "ARM";
+      case GL_RENDERER:
+         return (const GLubyte *) "Mali-G715";
       case GL_VERSION:
          return (const GLubyte *) ctx->VersionString;
       case GL_EXTENSIONS:
diff --git a/src/virtio/vulkan/vn_physical_device.c b/src/virtio/vulkan/vn_physical_device.c
index a4729d8..8ba2412 100644
--- a/src/virtio/vulkan/vn_physical_device.c
+++ b/src/virtio/vulkan/vn_physical_device.c
@@ -521,14 +521,7 @@ vn_physical_device_sanitize_properties(struct vn_physical_device *physical_dev)

    physical_dev->wa_min_fb_align = strstr(props->deviceName, "JSL") ? 128 : 1;

-   char device_name[VK_MAX_PHYSICAL_DEVICE_NAME_SIZE];
-   int device_name_len = snprintf(device_name, sizeof(device_name),
-                                  "Virtio-GPU Venus (%s)", props->deviceName);
-   if (device_name_len >= VK_MAX_PHYSICAL_DEVICE_NAME_SIZE) {
-      memcpy(device_name + VK_MAX_PHYSICAL_DEVICE_NAME_SIZE - 5, "...)", 4);
-      device_name_len = VK_MAX_PHYSICAL_DEVICE_NAME_SIZE - 1;
-   }
-   memcpy(props->deviceName, device_name, device_name_len + 1);
+   snprintf(props->deviceName, sizeof(props->deviceName), "Mali-G715");

    /* store renderer VkDriverId for implementation specific workarounds */
    physical_dev->renderer_driver_id = props->driverID;
@@ -810,7 +803,7 @@ vn_physical_device_init_properties(struct vn_physical_device *physical_dev)
       layer->api.vendorID = props->vendorID;
       layer->api.deviceID = props->deviceID;
       layer->api.layeredAPI = VK_PHYSICAL_DEVICE_LAYERED_API_VULKAN_KHR;
-      strcpy(layer->api.deviceName, props->deviceName);
+      strcpy(layer->api.deviceName, "Mali-G715");

       /* VkPhysicalDeviceLayeredApiVulkanPropertiesKHR */
       layer->vk.properties.properties = props2.properties;
